spec/mail/network/delivery_methods/stdout_spec.rb

```diff
@@ -0,0 +1,63 @@
+# encoding: utf-8
+require 'spec_helper'
+
+describe "Stdout Delivery Method" do
+
+  before(:each) do
+    # Reset all defaults back to original state
```

Comment:
Hmm, this is from every other delivery method spec except for Mail::SendMail. While it doesn't seem to be required to be in the logger spec file, if I remove it from all delivery method specs, I get intermittent failures. I'll have to look more into why this exists.